config_defaults_inc.php

```diff
@@ -5117,3 +5117,15 @@
  * the attachment is linked to the note.  Or 0 for disabling this feature.
  */
 $g_issue_activity_note_attachments_seconds_threshold = 3;
+
+#################################
+# Simple Report Screen settings #
+#################################
+
+/**
+ * Define the max Level Treshold  for users to present the simple entry screen
+ * Define the fileds to be displayed ( initial value is same as current set
+ * example : $g_simple_bug_report_page_fields = array(	'additional_info',	'attachments',	'category_id',);
+ */
+$g_simple_report_level = UPDATER;
+$g_simple_bug_report_page_fields = $g_bug_report_page_fields;
```

Comment:
Should be set to `'%bug_report_page_fields%'`
This will be less confusing if someone changes `$g_bug_report_page_fields` but not `$g_simple_bug_report_page_fields` in his own config/config_inc.php